apps/src/aichat/views/ChatWorkspace.tsx

```diff
@@ -98,4 +189,37 @@ const ChatWorkspace: React.FunctionComponent<ChatWorkspaceProps> = ({
   );
 };
 
+interface TestModelProps {
+  conversationContainerRef: React.RefObject<HTMLDivElement>;
+  items: ChatItem[];
+  showWaitingAnimation: () => React.ReactNode;
+}
+
+const TestModel: React.FunctionComponent<TestModelProps> = ({
```

Comment:
Minor: I know "Test Model" is the name of this tab but it looks like a little confusing to me as a component name in this file for some reason 🙂 maybe something like `ModelChat` or `ChatWithModel` or something similar? It's also possible that this inner component could be the same for both viewing chat history and the current chat since it's just rendering a list of chat items.